hw/formal/tools/jaspergold/parse-formal-report.py

```diff
@@ -164,8 +164,8 @@ def main():
         ''')
     parser.add_argument('--logpath',
                         type=str,
-                        default="fpv.log",
-                        help=('FPV log file path. Defaults to `fpv.log` '
+                        default="formal.log",
```

Comment:
I think you should skip the default and force the user to pass this arg, given that this script is invoked by dvsim.

Reply:
I see, that makes sense, thanks Sri for the suggestion.